packages/teleport/src/teleportContext.tsx

```diff
@@ -100,6 +100,8 @@ class TeleportContext implements types.Context {
       desktops: userContext.getDesktopAccess().list,
       nodes: userContext.getNodeAccess().list,
       activeSessions: userContext.getActiveSessionsAccess().list,
+      accessRequests: userContext.getAccessRequests().list,
+      newAccessRequest: userContext.getAccessRequests().create,
```

Comment:
It is strange to make this one based on `create`, but it **is** a screen to create access requests. Thoughts?

Reply:
makes sense